clippy_dev/src/fmt.rs

```diff
@@ -290,8 +290,14 @@ fn run_rustfmt(context: &FmtContext) -> Result<(), Error> {
         .filter_map(|entry| {
             let entry = entry.expect("failed to find tests");
             let path = entry.path();
-
-            if path.extension() != Some("rs".as_ref()) || entry.file_name() == "ice-3891.rs" {
+            if path.extension() != Some("rs".as_ref())
+                || path
+                    .components()
+                    .rev()
+                    .nth(1)
```

Comment:
This is a double ended iterator

```suggestion
                    .nth_back(1)
```